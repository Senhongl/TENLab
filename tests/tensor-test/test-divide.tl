a = [[1.1,2.0],[3.5,4.2]] / 0.5;
print(a);