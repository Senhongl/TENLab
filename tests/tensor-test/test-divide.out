 2.2000  4.0000
 7.0000  8.4000
[ CPUDoubleType{2,2} ]
